lib/src/uploader.dart

```diff
@@ -84,7 +85,7 @@ class Uploader {
     try {
       String responseBody = utf8.decode(response.bodyBytes);
       String requestBody = request.body;
-      if (code != 200) {
+      if (response.statusCode < 200 || response.statusCode > 300) {
```

Comment:
@kwent Would you please use the `code` variable defined a few lines above this one? Thanks!

Reply:
Done!